site/quickstart.md

```diff
@@ -1,159 +1,162 @@
+$ helm upgrade \
+  --reuse-values \
+  --set "domainNamespaces={sample-domains-ns1}" \
+  --wait \
+  sample-weblogic-operator \
+  kubernetes/charts/weblogic-operator
 
 ```
-          ./create-domain.sh
-          -i create-domain-inputs.yaml
-          -o /path/to/output-directory
+c.  Configure Traefik to manage Ingresses created in this namespace:
+```
+$ helm upgrade \
+  --reuse-values \
+  --set something about namespaces
 ```
 
-* For a domain home in image, use the sample in the Docker GitHub project.
-
-* Optionally, create a configuration overrides template and any additional Kubernetes secrets it needs (for example, to override the domain home configuration of a database URL, username, and password).
+## 5. Create a domain in the domain namespace.
 
-* Create a domain resource in the domain namespace.
-  * Specify the following information: domain UID, service account, secret name, the domain home details, and optionally, the configuration overrides template name.
+a.	Create a new image with a domain home by running the [`create-domain-in-image`](../kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/README.md) script.
+* First, create a directory for the files generated by the sample.
+* Then, copy the `inputs.yaml` file from the sample and modify it:
+  * Specify a `domainUID`:
+  * Specify the domain namespace: `sample-domains-ns1`
+  * Specify the number and type of servers:
 
-* Configure the operator to know about the domain.
-   * Edit the operator `values.yaml` file to add the namespace of the domain:
+b.	The sample also creates a domain resource YAML file.
+* Use `kubectl apply -f` on the YAML file to create the domain resource.
+    * This tells the operator to manage the new domain.
+    * The operator will notice and start up the servers in the domain.
 
-```
-$ helm update kubernetes/charts/weblogic-operator --name my-operator --namespace weblogic-operator-ns --values values.yaml --wait
-```
-
-* Configure the Traefik load balancer to manage the domain as follows:
-  * Create an Ingress for the domain in the domain namespace (it contains the routing rules for the domain):
+c.	Confirm that the operator started the servers for the domain by using `kubectl`.
+* Also, use `kubectl`to show that the domain resource was created.
+* Look at the operator log.
 
+d.	Create an Ingress for the domain, in the domain namespace, by using the [sample](../kubernetes/samples/charts/ingress-per-domain/README.md) Helm chart:
+* Use `helm install`, specifying the `domainUID` and domain namespace.
 ```
 $ cd kubernetes/samples/charts
 $ helm install ingress-per-domain --name domain1-ingress --value values.yaml
 ```
 
-(At this point, do they have a WebLogic Kubernetes deployment in a Kubernetes cluster? If so, we have to give them something to look at, to verify their results.)
+e.	Confirm that the load balancer noticed the new Ingress and is successfully routing to the domain's server pods.
```

Comment:
$ curl --silent http://${HOSTNAME}:30305/sample-domain1/

Reply:
Got it. Should I remove, Browse to a URL? IS there a sample application (in the domain home in image) that they can browse to?